tools/depends/native/ldid/Makefile

```diff
@@ -7,23 +7,16 @@ DEPS= ../../Makefile.include Makefile
 LIBNAME=ldid
 VERSION32=1.0.0
 VERSION64=2
-SOURCE32=$(LIBNAME)-$(VERSION32)
 SOURCE64=$(LIBNAME)-$(VERSION64)
-ARCHIVE32=$(SOURCE32).tar.gz
 ARCHIVE64=$(SOURCE64).tar.gz
 
 all: .installed-$(PLATFORM)
 
-$(TARBALLS_LOCATION)/$(ARCHIVE32):
-	cd $(TARBALLS_LOCATION); $(RETRIEVE_TOOL) $(RETRIEVE_TOOL_FLAGS) $(BASE_URL)/$(ARCHIVE32)
-
 $(TARBALLS_LOCATION)/$(ARCHIVE64):
 	cd $(TARBALLS_LOCATION); $(RETRIEVE_TOOL) $(RETRIEVE_TOOL_FLAGS) $(BASE_URL)/$(ARCHIVE64)
 
-$(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE32) $(TARBALLS_LOCATION)/$(ARCHIVE64) $(DEPS)
+$(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE64) $(DEPS)
 	rm -rf $(PLATFORM)/*; mkdir -p $(PLATFORM)
-	cd $(PLATFORM); $(ARCHIVE_TOOL) $(ARCHIVE_TOOL_FLAGS) $(TARBALLS_LOCATION)/$(ARCHIVE32)
-	mv $(PLATFORM)/ldid $(PLATFORM)/ldid32
 	cd $(PLATFORM); $(ARCHIVE_TOOL) $(ARCHIVE_TOOL_FLAGS) $(TARBALLS_LOCATION)/$(ARCHIVE64)
 	mv $(PLATFORM)/ldid $(PLATFORM)/ldid64
```

Comment:
Why 64?

Reply:
Fixed, also updated command call in codesign command.